app/lib/methods/loadMissedMessages.js

```diff
@@ -31,6 +31,8 @@ export default function loadMissedMessages(args) {
 		try {
 			const data = await load.call(this, { rid: args.rid, lastOpen: args.lastOpen });
 
+			await this.sdk.post('subscriptions.read', { rid: args.rid });
```

Comment:
This is wrong. You should not add another method here.
The goal for this file is to `load missed messages`.
Adding another method here is going to either add coupling or hide its purpose.